app/views/types/index.html.erb

```diff
@@ -0,0 +1,45 @@
+<h1 align='center'>Types</h1>
+<h2 align='center'>Yes, different types as well... Cause, why not?!</h2>
+
+<div style='text-align: center;'>
```

Comment:
dirty style attribute directly on the tag is dirty 😕
Can you rather call classes from css? 🐱‍👤